### ISSUE TITLE
python module fails on python3.11 and aarch64

### ISSUE THREAD

**Issue by polidore** [duckdb/duckdb#5672]
### What happens?

pip install duckdb builds from source. 

Fails on import: 
```
>>> import duckdb
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: Invalid Error: std::bad_alloc

Invalid Error: std::bad_alloc
```

### To Reproduce

1. Install on aarch64 and python3.11
2. Import

### OS:

Oracle Enterprise Linux 8.0

### DuckDB Version:

0.6.1

### DuckDB Client:

Python3.11

### Full Name:

Ben Polidore

### Affiliation:

personal

### Have you tried this on the latest `master` branch?

- [X] I agree

### Have you tried the steps to reproduce? Do they include all relevant data and configuration? Does the issue you report still appear there?

- [X] I agree

**Comment by polidore** [comment]
A couple of updates. 

Also fails with python3.9, so maybe an aarch64 issue? 

```
Python 3.9.7 (default, Apr 11 2022, 07:29:18)
[GCC 8.5.0 20210514 (Red Hat 8.5.0-10.0.1)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import duckdb
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: Invalid Error: std::bad_alloc
```

I tried installing latest dev, and the error is different, but it persists.
```
 pip3 install duckdb==0.6.2.dev447
Python 3.11.0 (main, Dec  4 2022, 15:04:56) [GCC 8.5.0 20210514 (Red Hat 8.5.0-15.0.1)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import duckdb
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: Out of Memory Error: Failed to allocate block of 2048 bytes
```

**Comment by Tishj** [comment]
Since you're building a dev version, I wonder if this is related to the compilation issue we're having on master
I use an M1 Mac to work on duckdb, `uname -m` reports `arm64`
As far I know that's the same as `aarch64`? So I don't think there's a python-specific issue with aarch64

**Comment by polidore** [comment]
I fixed it. The issue is OEL related. After googling a bit, I found some other projects that were having this issue. OEL had a default pagesize of 64kb for aarch64. In a recent update, they changed it to 4kb:

https://docs.oracle.com/en/operating-systems/uek/7/relnotes7.0/uek7.0-NewFeaturesandChanges.html#uek-arm-pagesize

After upgrading to that version of the kernel, duckdb works for me. Sorry for noise, but hopefully this helps someone else!
